File: AuthHub.Models/Organizations/CreateOrganizationRequest.cs
using System.ComponentModel.DataAnnotations;

namespace AuthHub.Models.Organizations
{
    public class CreateOrganizationRequest
    {
        [Required]
        public string Name { get; set; }
        [Required]
        [MinLength(8)]
        public string Password { get; set; }
        [Required]
        [MinLength(8)]
        public string ConfirmPassword { get; set; }
        [Required]
        [EmailAddress]
        public string Email { get; set; }
    }
}

File: authhub-ui/src/app/models/requests/CreateOrganizationRequest.ts
export class CreateOrganizationRequest{
    Name: string | null | undefined;
    Password: string | null | undefined;
    ConfirmPassword: string | null | undefined;
    Email: string | null | undefined;

    constructor(init?: Partial<CreateOrganizationRequest>){
        Object.assign(this, init)
    }
}